src/devices/src/virtio/balloon/device.rs

```diff
@@ -0,0 +1,920 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cmp;
+use std::io::Write;
+use std::result;
+use std::result::Result;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+use std::time::Duration;
+use timerfd::{ClockId, SetTimeFlags, TimerFd, TimerState};
+
+use ::utils::eventfd::EventFd;
+use logger::{error, Metric, METRICS};
+use versionize::{VersionMap, Versionize, VersionizeResult};
+use versionize_derive::Versionize;
+use virtio_gen::virtio_blk::*;
+use vm_memory::{Address, ByteValued, Bytes, GuestAddress, GuestMemoryMmap};
```

Comment:
nit:
Since Rust 2018 external crates should be imported like `use ::logger::bla` for clarity.

Reply:
Changed it to match the Rust 2018 style.